README.rst

```diff
@@ -2,21 +2,28 @@ intel-iot-refkit
 ################
 
 This README file contains information on setting up, building, and booting
-Iot Reference OS Kit for Intel(r) Architecture project.
+the IoT Reference OS Kit for Intel(r) Architecture project.
+
+.. note::
+
+   Only Linux development hosts are currently support, as documented
```

Comment:
sigh... I shouldn't do these patches when I'm distracted.  Sorry about the spelling error.